test/JDBC/upgrade/13_4/preparation/bbf_view_def_upgrade-vu-prepare.sql

```diff
@@ -0,0 +1,10 @@
+CREATE VIEW view_3135_upd AS SELECT 1;
```

Comment:
Use filename_xxx as object name at all places to allow unique object names

Reply:
Fixed